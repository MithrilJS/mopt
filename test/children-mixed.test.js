"use strict";

var code = require("./lib/code");

describe("Children", () => {
    describe("Mixed Children", () => {
        it("should support mixed array and literal children", () =>
            expect(
                code(`m("0", [ 1 ], 2)`)
            )
            .toMatchSnapshot()
        );
        
        it("should support multiple arrays of children", () =>
            expect(
                code(`m("0", [ 1 ], [ 2 ], [ 3 ])`)
            )
            .toMatchSnapshot()
        );
    });
});
