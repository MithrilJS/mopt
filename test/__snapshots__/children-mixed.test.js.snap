// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Children Mixed Children should support mixed array and literal children 1`] = `"m.vnode(\\"0\\",undefined,undefined,[m.vnode(\\"[\\",undefined,undefined,[m.vnode(\\"#\\",undefined,undefined,1,undefined,undefined)],undefined,undefined),m.vnode(\\"#\\",undefined,undefined,2,undefined,undefined)],undefined,undefined);"`;

exports[`Children Mixed Children should support multiple arrays of children 1`] = `"m.vnode(\\"0\\",undefined,undefined,[m.vnode(\\"[\\",undefined,undefined,[m.vnode(\\"#\\",undefined,undefined,1,undefined,undefined)],undefined,undefined),m.vnode(\\"[\\",undefined,undefined,[m.vnode(\\"#\\",undefined,undefined,2,undefined,undefined)],undefined,undefined),m.vnode(\\"[\\",undefined,undefined,[m.vnode(\\"#\\",undefined,undefined,3,undefined,undefined)],undefined,undefined)],undefined,undefined);"`;
